cerr/errors.go

```diff
@@ -119,6 +119,10 @@ var (
 	ErrQueryExecutionFailed = newErr("E10048", codes.FailedPrecondition, http.StatusBadRequest, "query failed")
 	// Returned when the account has not setup a personal tenant yet
 	ErrPersonalTenantRequired = newErr("E10049", codes.FailedPrecondition, http.StatusBadRequest, "personal tenant required")
+	// Returned if a policy builder id is not found in the database
+	ErrPolicyBuilderNotFound = newErr("E10050", codes.NotFound, http.StatusNotFound, "policy builder not found")
+	// Returned if a policy builder id is invalid
+	ErrInvalidPolicyBuilderID = newErr("E10051", codes.InvalidArgument, http.StatusBadRequest, "invalid policy builder id")
```

Comment:
Isn't the policy builder id something that we generate? when could this be invalid?

Reply:
When it's provided by the user on delete request

Reply:
Why not return `ErrPolicyBuilderNotFound` in that case?

Reply:
Do be able to differentiate between the case when a PolicyBuilder ID is not in the database & a PolicyBuilder ID is incorrect